src/VisualStudio/Core/Def/EditorConfigSettings/SettingsEditorFactory.cs

```diff
@@ -175,5 +175,22 @@ public int MapLogicalView(ref Guid rguidLogicalView, out string? pbstrPhysicalVi
                 return VSConstants.E_NOTIMPL;   // you must return E_NOTIMPL for any unrecognized rguidLogicalView values
             }
         }
+
+        public object GetDocumentData(uint grfCreate, string pszMkDocument, IVsHierarchy pHier, uint itemid)
```

Comment:
@davidwengier I need to test this out but I assume its ok to just throw `NotImplementedException` for these?

Reply:
Yep, the runtime should translate this to the right HRESULT. I suspect returning null would work too (never did merge my implementation of IVsEditorFactory4 🤫)